manage_breast_screening/notifications/management/commands/send_message_batch.py

```diff
@@ -58,15 +61,19 @@ def handle(self, *args, **options):
                 MessageBatchHelpers.mark_batch_as_sent(message_batch, response.json())
                 self.stdout.write(f"{message_batch} sent")
             else:
-                self.mark_batch_as_failed(message_batch)
+                self.mark_batch_as_failed(message_batch, response)
                 self.stdout.write(
                     f"Failed to send batch. Status: {response.status_code}"
                 )
         except Exception as e:
             raise CommandError(e)
 
-    def mark_batch_as_failed(self, message_batch: MessageBatch):
-        message_batch.status = "failed"
+    def mark_batch_as_failed(self, message_batch: MessageBatch, response: Response):
+        message_batch.notify_errors = response.json()["errors"]
```

Comment:
I wanted to manually check the response is in this shape when we get it back from NHS Notify directly,  so used CMAPI to make a request and get a 401 [ via this branch ](https://github.com/NHSDigital/dtos-communication-management/pull/new/test-notify-error) and it looks correct!
Just an FYI if we do decide to still go through CMAPI  instead of direct to Notify I think CMAPI wraps it in a `response` key so it would be response.json()["response"]["errors"]